source/docs/software/vision-processing/roborio/using-multiple-cameras.rst

```diff
@@ -113,12 +160,20 @@ If you're using some other dashboard, you can change the camera used by the came
          prevTrigger = joy1.GetTrigger();
        }
 
+    .. code-block:: python
+
+       # Setting the source directly via joystick isn't possible in Python, you
```

Comment:
The previous example is only really applicable with SmartDashboard. Can NT be used to communicate between the two processes?

Reply:
The previous example uses `NetworkTableInstance.getDefault().getTable("").getEntry("CameraSelection")`... which is NetworkTables? So yes.

Reply:
I guess my point is that it isn't so obvious what the code would look like. There needs to be NT code in the joystick section that isn't needed in the previous example.

Reply:
I don't understand. The code is exactly the example above. See line 78-90 of this file. It's using a joystick to set a NetworkTables value, which controls the camera.